config/routes.rb

```diff
@@ -4,8 +4,17 @@
   # Defines the root path route ("/")
   # root "articles#index"
   root 'users#index'
-  get '/users', to: 'users#index'
-  resources :users, only: [:index, :show] do
-    resources :posts, only: [:index, :show]
-  end
+  # get '/users', to: 'users#index'
+  # resources :users, only: [:index, :show] do
+  #   resources :posts, only: [:index, :show]
+  # end
```

Comment:
- [ Required ] I think it would be nice if you remove any piece of commented code in this file, commented code usually causes confusion and is considered as a sign of code that is not clean. It also makes your project look unprofessional, please rectify this.